sources/Interop/Windows/um/winnt/TEB.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/winnt.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public partial struct TEB
```

Comment:
Opaque struct? Should this have been a handle?

Reply:
Corrected in a future PR.